app/src/main/kotlin/at/bitfire/davdroid/sync/adapter/SyncAdapterImpl.kt

```diff
@@ -120,9 +122,12 @@ class SyncAdapterImpl @Inject constructor(
         // As a defensive workaround, we can cancel specifically this still pending sync only
         // See: https://github.com/bitfireAT/davx5-ose/issues/1458
         if (Build.VERSION.SDK_INT >= 34) {
-            logger.fine("Android 14+ bug: Canceling forever pending sync adapter framework sync request for " +
-                    "account=$account authority=$authority upload=$upload")
-            syncFrameworkIntegration.cancelSync(account, authority, extras)
```

Comment:
Can we just use the `accountOrAddressBookAccount` argument instead of `account` and instead of building it from `account` again?

Reply:
Oh! I completely overlooked that. It should work then. I will check and change :+1:

Reply:
@ArnyminerZ  Can you verify it still works for you too?